### ISSUE TITLE
Export queue

### ISSUE THREAD

**Issue by rubydesign** [xeolabs/xeogl#242]
I'm basically done with my project, it came out quite nice I think.

I have about 1000 pieces that make up the structure of a house(that can be resized) . As it is difficult to see the whole, I animate the pieces into existence.

Having tried different approaches and having read how you do it, I copied the queue that you use as task queue. Using the task queue did not quite work and does not give enough control over timing.

Anyway, long story short. Since the queue is there, it may as well be exposed, as it is useful.
